credentialz/credentialz.proto

```diff
@@ -429,6 +429,8 @@ enum KeyType {
   KEY_TYPE_ED25519 = 3;
   KEY_TYPE_RSA_2048 = 4;
   KEY_TYPE_RSA_4096 = 5;
+  KEY_TYPE_RSA_3072 = 6;
```

Comment:
What's the reason to support 3072? "get all the pokemons" or because of some actual security benefit over 4096? (my recollection is that 3072 is barely considered safe today, and with post-quantum coming to eat our pokemons this is not safe 'shortly'.)

Reply:
As we already support 2048 so not supporting 3072 because of security consideration doesn’t seem right to me.
With quantum computing we can’t say 4096 will make a difference when compared to 3072. And it’s a proto so we should let users have more control on the keys that they want to use. In addition, 3072 is a FIPS allowed so I think we should have this.